docker/README.md

```diff
@@ -0,0 +1,15 @@
+# Using docker-compose
+
```

Comment:
are you missing the example commands here?

Reply:
No, there are some below after the prerequisite of creating a `.env` from one of the sample files; do you wish to see more examples?